packages/@aws-cdk/aws-eks/lib/cluster-resource-handler/cluster.ts

```diff
@@ -137,10 +137,17 @@ export class ClusterResourceHandler extends ResourceHandler {
     }
 
     if (updates.updateLogging || updates.updateAccess) {
```

Comment:
@watany-dev an error should be thrown when `updates.updateLogging` and `updates.updateAccess` are both true, as updating both logging configuration and resources vpc configuration results in an error from the EKS API (even though it is not documented in [EKS UpdateClusterConfig API](https://docs.aws.amazon.com/eks/latest/APIReference/API_UpdateClusterConfig.html))

Reply:
It seems I did not know the correct specifications. I will try to re-write it again with that useful information. Thanks.